testscript/doc.go

```diff
@@ -46,11 +46,11 @@ create in the script's temporary file system before it starts executing.
 As an example:
 
 	# hello world
-	exec cat hello
-	stdout "hello world\n"
+	exec cat hello.text
```

Comment:
Minor preference for `.txt` ...

Reply:
I just copied what's in testdata. The point of not using `.txt` is that the test would overwrite itself :)

I slightly prefer this over a filel called just `hello`, because that's less obviously a file.

Reply:
> The point of not using .txt is that the test would overwrite itself :)

No, because this is all run within a temp directory `$WORK`. So the txtar is expanded in `$WORK` and your initial working directory is `$WORK`

Agreed having a suffix is better than just `hello`.

Reply:
Gotcha. Even then, I'd say it could be confusing for a hello world test to use a temporary file that happens to be named just like the test script iself.

Ultimately this is up to @rogpeppe - but whatever we decide, the two scripts should be identical.

Reply:
How about just using "hello" as a file name?

Reply:
Three people, three ways of naming one silly file :)

Roger, did you see our previous comments about no filename extension? If you still prefer `hello`, I'll just change it to that.